doc/source/user_guide/numpyro_refitting_xr_lik.ipynb

```diff
@@ -2090,11 +2091,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We are now missing the `log_likelihood` group because we have not used the `log_likelihood` argument in `idata_kwargs`. We are doing this to ease the job of the sampling wrapper. Instead of going out of our way to get Stan to calculate the pointwise log likelihood values for each refit and for the excluded observation at every refit, we will compromise and manually write a function to calculate the pointwise log likelihood.\n",
+    "We are now missing the `log_likelihood` group because we have not used the `log_likelihood` argument in `idata_kwargs`. We are doing this to ease the job of the sampling wrapper. Instead of going out of our way to get NumPyro to calculate the pointwise log likelihood values for each refit and for the excluded observation at every refit, we will compromise and manually write a function to calculate the pointwise log likelihood.\n",
     "\n",
-    "Even though it is not ideal to lose part of the straight out of the box capabilities of PyStan-ArviZ integration, this should generally not be a problem. We are basically moving the pointwise log likelihood calculation from the Stan code to the Python code, in both cases we need to manually write the function to calculate the pointwise log likelihood.\n",
```

Comment:
The same goes for `PyStan-ArviZ integration`.

Reply:
Done but actually I never changed any code but rather the docs. I mean onlly the text content was changed, so all the test should still run right?(just asking)

Reply:
Yeah, there is some other issue with tests, nothing is wrong with your PR. 
Thank you so much for your PR. I will merge it as soon as the other gets merged. Feel free to explore other issues as well.

Reply:
So um, I didn't do anything wrong with this PR right and I'm good to go?(I'm mean was everything I did correct?)

Reply:
Yes. Thank you :))